beaconclient/multi_beacon_client.go

```diff
@@ -94,15 +95,15 @@ func NewMultiBeaconClient(log *logrus.Entry, beaconInstances []IBeaconInstance)
 		client.ffAllowSyncingBeaconNode = true
 	}
 
-	if os.Getenv("BROADCAST_VALIDATION") != "" {
-		broadcastValidationStr := os.Getenv("BROADCAST_VALIDATION")
-		broadcastValidation, ok := parseBroadcastValidationString(broadcastValidationStr)
+	broadcastModeStr := os.Getenv("BROADCAST_MODE")
+	if broadcastModeStr != "" {
+		broadcastMode, ok := parseBroadcastModeString(broadcastModeStr)
 		if !ok {
-			msg := fmt.Sprintf("env: BROADCAST_VALIDATION: invalid value %s, leaving to default value %s", broadcastValidationStr, client.ffBroadcastValidation.String())
+			msg := fmt.Sprintf("env: BROADCAST_MODE: invalid value %s, leaving to default value %s", broadcastModeStr, client.broadcastMode.String())
 			client.log.Warn(msg)
 		} else {
-			client.log.Info(fmt.Sprintf("env: BROADCAST_VALIDATION: setting validation to %s", broadcastValidation.String()))
-			client.ffBroadcastValidation = broadcastValidation
+			client.log.Info(fmt.Sprintf("env: BROADCAST_MODE: setting validation to %s", broadcastMode.String()))
```

Comment:
Should this say "mode" instead of "validation"?
```suggestion
			client.log.Info(fmt.Sprintf("env: BROADCAST_MODE: setting mode to %s", broadcastMode.String()))
```